docs/release/release-tasks.md

```diff
@@ -1,547 +0,0 @@
-7. **For latest stable releases** Index the most recent CRDs in the release by navigating to `https://doc.crds.dev/github.com/kubernetes-sigs/cluster-api@<CURRENT_RELEASE>` 
-
-Additional information:
-
-* [Versioning documentation](./../../CONTRIBUTING.md#versioning) for more information.
-* Cutting a release as of today requires permissions to:
-  * Create a release tag on the GitHub repository.
-  * Create/update/publish GitHub releases.
-
-#### [Optional] Public release session
-   1. Host a release session over a public zoom meeting.
-   2. Record the session for future reference and transparency.
-   3. Use release process-related waiting periods as a forum for discussing issues/questions.
-   4. Publish the recording on YouTube channel.
-
-#### [Optional] [Track] Bump the Cluster API apiVersion
-
-**Note** This should only be done when we have to bump the apiVersion of our APIs.
-
-1. Add new version of the types:
-    1. Create new api packages by copying existing packages.
-    2. Make sure webhooks only exist in the latest apiVersion (same for other subpackages like `index`).
-    3. Add conversion and conversion tests.
-    4. Adjust generate targets in the Makefile.
-    5. Consider dropping fields deprecated in the previous apiVersion.
-2. Update import aliases in `.golangci.yml`.
-3. Switch other code over to the new version (imports across the code base, e.g. controllers).
-    1. Add all versions to the schema in the `main.go` files.
-4. Add types to the `PROJECT` files of the respective provider.
-5. Add test data for the new version in `test/e2e/data/{infrastructure-docker,shared}` (also update top-level `.gitignore`).
-6. Update `docker.yaml`, make sure all tests are successful in CI.
-
-#### [Optional] [Track] Bump the Kubernetes version
-
-1. Create an issue for the new Kubernetes version via: [New Issue: Kubernetes bump](https://github.com/kubernetes-sigs/cluster-api/issues/new/choose).
-2. Track the issue to ensure the work is completed in time.
-
-#### [Optional] Track Release and Improvement tasks
-
-1. Create an issue for easier tracking of all the tasks for the release cycle in question.
-   <br>Prior art: [Tasks for v1.6 release cycle](https://github.com/kubernetes-sigs/cluster-api/issues/9094)
-2. Create a release improvement tasks [GitHub Project Board](https://github.com/orgs/kubernetes-sigs/projects/55) to track 
-   the current status of all improvement tasks planned for the release, their priorities, status (i.e `Done`/`In Progress`)
-   and to distribute the work among the Release Team members.
-
-   **Notes**:
-   * At the beginning of the cycle, Release Team Lead should prepare the improvement tasks board for the ongoing release cycle.
-     The following steps can be taken:
-      - Edit improvement tasks board name for current cycle (e.g. `CAPI vX.Y release improvement tasks`)
-      - Add/move all individual missing issues to the board
```

Comment:
I think the replacement for all of these are missing?

Reply:
@chrischdi Thank you for pointing that out, I missed those points. I have added them. I will add a summary of changes below 👍